### PR TITLE
Fix/ltc redeem

### PR DESCRIPTION
**Pull request by envin3**
- Fix ltc redeem bug upgrading to ptokens.js 1.7.1
- Redirect to email contact for pBTC on gnosis
- update ultra endpoint

### CODE REVIEW

File: src/settings/index.js [hunk 1]
```diff
@@ -110,10 +110,10 @@ const settings = {
       },
       ultra: {
         chainId: 'a9c481dfbc7d9506dc7e87e9a137c931b0a9303f64fd7a1d08b8230133920097',
-        host: 'https://corsproxy.ptokens.io/v1/?apiurl=http://ultra-mainnet-1.ptokens.io:8888',
+        host: 'https://corsproxy.ptokens.io/v1/?apiurl=https://api.ultra.eossweden.org',
```
Comment (oliviera9) [inline src/settings/index.js:113]: Does it make sense to maintain corsproxy?

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "plain-dapp-update",
   "private": true,
-  "version": "1.20.2",
+  "version": "1.21.0",
```
Comment (oliviera9) [inline package.json:4]: I see this release as a fix. Isn't it?

Comment (envin3) [reply package.json:4]: The PR started as a fix, but
https://github.com/pnetwork-association/ptokens-dapp/pull/69/commits/f3e4ec2ff72f0a2677f615e1434e9bd649c24369
is actually a feat, should I rename the PR?

File: src/settings/index.js (hunk 1 above)
Comment (envin3) [reply src/settings/index.js:113]: It is there also for all the others eos like chains. Should it be removed from all of them?

Comment (oliviera9) [reply src/settings/index.js:113]: Honestly I don't know why it's used here. Maybe it was necessary to use our node.

Comment (envin3) [reply src/settings/index.js:113]: It will be removed in the next release

**Pull request merged**